docs/mllib-clustering-pic.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: global
+title: Clustering - MLlib
+displayTitle: <a href="mllib-guide.html">MLlib</a> - Power Iteration Clustering
+---
+
+* Table of contents
+{:toc}
+
+
+## Power Iteration Clustering
+
+Power iteration clustering is a scalable and efficient algorithm for clustering points given pointwise mutual affinity values.  Internally the algorithm:
+
+* computes the Gaussian distance between all pairs of points and represents these distances in an Affinity Matrix
```

Comment:
Doc needs update. We assume that the input is a graph with precomputed pairwise similarities/distances.